src/main.js

```diff
@@ -110,7 +110,7 @@ module.exports.apifyGoogleAuth = async ({ scope, tokensStore, credentials, googl
         console.log(information);
 
         const server = http.createServer((req, res) => {
-            if (req.url.includes('/authorize')) {
+            if (/[?&]code=/.test(req.url)) {
```

Comment:
1. Would just a comment with example URL with the code
2. I guess we can do the check with URL class and `code` param

Reply:
Yeah, makes sense. So let's just parse the URL